File: w08/d02/Victor_Rodriguez/cars_rails_app/java_objects/more_objects.js
var person = {
  
  name: "Spike Spiegel"
  height: "6'2\""
}

console.log(spike.name);
console.log(spike.height);

console.log();
var jack = Object.create(spike);

console.log(spike);
console.log(jack);

var Person ={
  sayHello: function() {
    return "Yo! My name be"
 + this.name + ".";
   }
   isOlder: function(other)
    return this.age > other.age;
   }
};

var ed = Object.create(Person)
ed.age = 25

var jet = Object.create(Person);
jet.age = 43;

console.log(jet.isOlder(ed));

File: w08/d02/Emmanuel_Tucker/more.objects.rb
class Person
     def sayHello
     return "Yo! My name is #{@name}."
          end

          def  is0lder(other)
          return @age > other.age
               end

          def age ()
          return @age
          end


          def age = (new_value)
          @age = new_value

     end
end

ed = Person.new()

ed.age = 15

jet = Person.new()
jet.age = 43


puts jet.is0lder(ed)
